gnucash/gtkbuilder/dialog-import.glade

```diff
@@ -1122,21 +1142,6 @@
     <child>
       <placeholder/>
     </child>
-    <child>
```

Comment:
Why are you removing the Show source account button?

Reply:
Bob,
I wasn't aware any of that had changed. This was a recommit of some changes from August last year that got lost, most likely as a result of my inexperience with git and github. I copied the changed file into the commit rather than reapplying the required changes so it is likely I overwrote some changes since my original commit.  I will reapply the required changes to a current copy of the file and resubmit it. I have to make a change John has suggested as well so I will incorporate that in the next commit as well.

David

Reply:
Bob,

That was indeed what had happened. I only had to change the treeview and treeview selection settings to enable multiple selection so I have copied the current dialog-import.glade from the maint branch into my feature branch and then modified it.  Will test it, commit and then do another pull request.

David

Reply:
Dave,
I still think it does not look right, had a look at your branch and there are now two commits, the first one with all the removal and a second one with John's change request with the glade changes but no sign of all the required changes in the 'c' source file. You can probably fix this with some git commands but it may be simpler to update your git maint branch so it is current, create a new branch for your changes and then make your changes there.
Bob